src/Pos/Block/Network/Retrieval.hs

```diff
@@ -186,11 +227,21 @@ workerHandle getPeers sendActions (peerId, headers) = do
     uselessFormat =
         "Chain of headers from " %shortHashF % " to " %shortHashF %
         " is useless for the following reason: " %stext
+    oldFormat =
+        "Chain of headers from " %shortHashF % " to " %shortHashF %
+        " is invalid for the following reason: the newest header is from"%
```

Comment:
Besides I've asked to move this message into `CHsUseless`, i'll emphasize once more: it's not invalid from the design perspective. Invalid stays for "broken" so we can effectively ban neighbor if we got something malformed from her. Useless info is just skipped.